### ISSUE TITLE
Guide View Controllers Objective-C -> Swift Conversion

### ISSUE THREAD

**Issue by NinjaSnail42** [suicidesafetyplan/safetyplan-ios#13]
Convert the Guide View Controllers to Swift.

- [X] GuideCopingViewController

- [X] GuideSuicidePreventionViewController

- [ ] GuideSafetyPlanViewController

- [X] GuideResourcesViewController

- [X] GuideOverviewViewController

- [X] GuideRecoveringViewController

**Comment by shayneptorres** [comment]
@NinjaSnail42 Just noticed that you had `EditWarningSignsViewController` on your list of views to update. I had already started updating this since it was part of the SafetyPlan view controllers. Were you still planning on working on it?

**Comment by NinjaSnail42** [comment]
Yeah sorry, I was listing off the Objective-C files that were in the "Guide" group, but it looks like the .m file is in the "Guide" group, and the header file is in the "Safety Plan" group. You can continue to update it I have not started working on it.

**Comment by NinjaSnail42** [comment]
I just wanted to mention that I was still working on this. With everything that has been going on, I have suddenly had a lot more to do, and I have some issues now with Xcode not wanting to show the Swift file in the Assistant editor after I tried to merge....

**Comment by eddielement** [comment]
That's so odd! Thanks for the update though :)

**Comment by igroti** [comment]
Can I try working on this? Is it still wanted/not done yet?

**Comment by eddielement** [comment]
@DaniSpringer Go ahead! I will keep updating the app :)

**Comment by igroti** [comment]
@eddielement I just opened the project and it has several warnings. I can fix those, if you would like to merge that (and can all those fixes be in a single branch or would you need them separated)?

Attaching:

<img width="401" alt="Screenshot of xcode issues list" src="https://github.com/suicidesafetyplan/safetyplan-ios/assets/19246409/be5ed4f1-cd24-4dc1-8f8a-ab25376e743d">

**Comment by igroti** [comment]
@eddielement also: might I ask what RealmSwift is for, and whether it's necessary? If it's the only pod, and can be done without, we might be able to get rid of pods altogether in the project 🍾

**Comment by eddielement** [comment]
@DaniSpringer All one branch would be great! RealmSwift is the database that is used to store the safety plan, but we could consider shifting it over to Swift Package Manager. :) https://realm.io/realm-swift/

**Comment by igroti** [comment]
@eddielement new spm branch much better, just note that the Xcode console printed something about a new version of Realm being available, even while running the app via the spm branch.

Thanks
